src/comments.js

```diff
@@ -0,0 +1,86 @@
+  p.classList.add('name');
+  p.innerHTML = mealName;
+  const category = document.createElement('p');
+  category.innerHTML = `This meal belongs to <strong>${meal.strCategory}</strong> Category`;
+  const recipe = document.createElement('p');
+  const Instructions = meal.strInstructions.substring(0, 130);
+  recipe.innerHTML = `<strong>Instructions for making</strong> ${Instructions}  =>FIND MORE AT  <a target="_blank" href=${meal.strYoutube}>Youtube</a>`;
+  modalBody.appendChild(img);
+  const div = document.createElement('div');
+  div.classList.add('items');
+  div.appendChild(p);
+  div.appendChild(category);
+  div.appendChild(recipe);
+  const headingFour = document.createElement('h4');
+  const comments = await displayComment(id);
+  headingFour.innerHTML = `comments (${commentsCounter(comments)})`;
+  div.appendChild(headingFour);
+  comments.forEach((comment) => {
+    const p = document.createElement('p');
+    p.classList.add('comment-p');
+    p.innerHTML = `${comment.creation_date} ${comment.username}: ${comment.comment}`;
+    return div.appendChild(p);
+  });
+  modalBody.appendChild(div);
+  const form = document.createElement('form');
+  form.setAttribute('id', `${itemId}`);
+  form.setAttribute('class', 'form');
+  form.innerHTML = `<div class="mb-3">
+               <h3>Add a comment</h3>
+               <input type="text" class="form-control  mt-3 py-2" id="name" placeholder="Your Name">
+             </div>
+             <div class="mb-3">
+               <textarea type="text" placeholder="Your Insights" class="form-control py-2" id="comment"></textarea>
+             </div>
+             <button type="submit" id="button" class="btn btn-dark right button">Comment</button>`;
+  modalBody.appendChild(form);
+  form.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    const username = document.querySelector('#name');
+    const comment = document.querySelector('#comment');
+    const formData = {
+      item_id: e.target.id,
+      username: username.value,
+      comment: comment.value,
+    };
+    if (username.value === '') return addAlert('name', 'danger');
+    if (comment.value === '') return addAlert('comment', 'danger');
+    if (await addComment(formData)) {
+      comment.value = '';
+      username.value = '';
+      addAlert('Comment added succesfully', 'success');
+      const comments = await displayComment(id);
+      headingFour.innerHTML = `comments (${commentsCounter(comments)})`;
+      div.replaceChild(headingFour, headingFour);
+      [...div.children].forEach((child) => {
+        if (child.className === 'comment-p') {
+          div.removeChild(child);
+        }
+      });
+      comments.forEach((comment) => {
+        const newp = document.createElement('p');
+        newp.innerHTML = `${comment.creation_date} ${comment.username}: ${comment.comment}`;
+        return div.appendChild(newp);
+      });
+    }
+  });
+};
+
+export default addModal;
```

Comment:
- [ ] Every time we click on the comment button we will get a comment pop up for the specific card that contains the image and description to allows us to add a comment instead of displaying many cards in one pop up comment, this is means that if I need to add a comment on the `Beef Lo Mein` the pop up message will appear on the screen with `Beef Lo Mein's description` card only.

`Make sure a user adds a comment on the specific card he chose instead of displaying all items in one comment pop up`.

- [ ] The closing button for the comment pop up redirect us to the broken link, kindly fix it.

![closebtn](https://user-images.githubusercontent.com/55635977/132805813-70a7b76d-8bb5-405b-82da-c71a5d45ca38.png)

Reply:
`To the next reviewer.`

`The above error seems to be an issue with gh-pages because it working on his locally computer. `